src/components/KeyboardShortcuts/index.tsx

```diff
@@ -97,6 +120,32 @@ export class KeyboardShortcutsBase extends React.Component<Props> {
             }),
           );
           break;
+        case 'n':
+          if (compareInfo) {
+            dispatch(
+              _goToRelativeDiff({
+                currentAnchor,
+                diff: compareInfo.diff,
+                pathList,
+                position: RelativePathPosition.next,
+                versionId,
+              }),
+            );
+          }
```

Comment:
I'd like to at least see a warning in the console for the `else` branch here. Ideally, it should pop up a message on the screen but we don't have a framework for that. I think it would be confusing when pressing `p` does nothing on the browse page.

Reply:
Ok, I added this. Do you think we also need a test for this (which would involve needing to inject a fake logger), or is this the kind of thing we don't need to bother testing as it is just a warning?

Reply:
I don't see any value in adding a test with a fake logger and asserting it gets called correctly. TS already handles this. Actually, I think we could all stop adding these types of tests.